cypress/e2e/aboutpage.cy.ts

```diff
@@ -0,0 +1,37 @@
+describe("About page tests", () => {
+  beforeEach(() => {
+    cy.visit("/about",  { timeout: 30000 });
+  });
+
+  it("About header renders", () => {
+    cy.get('[data-cy="about-header-header"]')
+      .should("be.visible")
+      .and("contain", "Transnational Job Listing Channel");
+    cy.get('[data-cy="about-header-body"]')
+      .should("be.visible")
+      .and("contain", "So many jobs available, all you have to do is keep up with  our posts. Check below for recent job openings.");
+  });
+
+  it("About banner renders", () => {
+    cy.get('[data-cy="about-banner-header"]')
+      .should("be.visible")
+      .and("contain", "Have a question?");
```

Comment:
@marktnoonan I like how your suggestion avoids a new test id, but am concerned about a check for h4, when it really shouldn't be an h4 from an a11y perspective. That is, headings should only increase by one level at a time and this skips from h1 to h4. A check for h4 suggests otherwise. I suppose I could implement your suggestion and log an issue about the skipped heading level (which would require an app fix and test fix). What do you think?

Reply:
@dgodongm lol I didn't check the heading nesting my bad, I assumed it was suitable. Let's not specify the h4 here then, since we know it's wrong. But please do make a new issue if you have the time so we can correct the heading nesting (and maybe fix any style side effects?) and then we can update the test as well.

I also would be totally fine if you just fix the nesting in this PR so that this heading is at the correct level, if it doesn't look like it will cascade into a whole bunch of document structure fixes.

Reply:
@marktnoonan I'll log an issue. I think it will be a bit more involved to fix the heading issues correctly so heading levels of re-usable components (molecules) can be set dynamically depending on context. There's also the issue of there being multiple h1 headers on the About page (from hard-coded h1 elements in molecules)

Reply:
Logged #166 and #167 for the a11y issues